build/stf-run-ci/tasks/setup_stf_local_build.yml

```diff
@@ -1,5 +1,7 @@
 ---
-# NOTE: the split filter cuts the image path (quay.io:443/infrawatch/container_image:tag_name) on the colon. Field :-1 (everything but the final field) is the image path, field -1 (final field) is the image tag
+# WARNING: generation of bundles is not idempotent from the point of being able
+# to use the generate_bundle_<name> content for use in other places
```

Comment:
I actually adjusted a fair amount of logic in stf-run-ci that while this statement is true, it actually doesn't fail any more. See pre-clean.yml for clean up of bundle and index builds. They are pretty quick to build, when generate_bundle is executed, we'll rebuild our bundles and index image. Probably useful for iterative testing of changes to the bundle layout/content?

There isn't really an easy way to get the current bundle version without just getting the new json from generate_bundle, so it's just easier to add the re-gen step each time it is run. Not sure how this affects the system if you've already subscribed and gotten a working environment.